custom/enikshay/case_utils.py

```diff
@@ -597,6 +601,11 @@ def iter_all_active_person_episode_cases(domain, case_ids):
         if episode_case.closed:
             continue
 
+        if sector == 'private' and episode_case.get_case_property('enrolled_in_private') != 'true':
```

Comment:
Case properties and values may also be good candidates to be a constant. Is there a place in the code that lists assumptions about app/case structure like this?